pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -574,9 +575,13 @@ def _multilingual_m_text(self, parent, name):
         self._multilingual_text(parent, name)
 
     def _multilingual_text(self, parent, name):
+        fallback_language = Config.get('default_language')
```

Comment:
We should not do it like that, because this way the Configuration will be read for every element of text (it should be read only once).